hirola/front/views.py

```diff
@@ -386,7 +400,9 @@ def search_view(request):
                         phone_color_quantity__is_in_stock=True,
                         phone_color_quantity__quantity__gte=1))
         results = list(set(results))
-        args = {"results": results, "instructions": False}
+        args = {"results": results, "instructions": False,
```

Comment:
What is the purpose fo the instructions argument being passed?

Reply:
I honesttly dont know, you are the one who put it, should I remove it